indico/modules/rb_new/client/js/common/rooms/RoomEditModal.jsx

```diff
@@ -0,0 +1,253 @@
+            this.renderFormGroup(
+                this.renderFieldList([{
+                    name: 'longitude',
+                    label: 'Longitute',
+                }, {
+                    name: 'latitude',
+                    label: 'Latitude'
+                }]),
+                'location_coordinates'
+            ),
+            ...this.renderFieldList([{
+                name: 'surfaceArea',
+                label: 'Surface Area (m2)'
+            }, {
+                name: 'maxAdvanceDays',
+                label: 'Maximum advance time for bookings (days)'
+            }])
+        ], 'col2'),
+        this.renderGridColumn([
+            this.renderHeader('Options'),
+            ...this.renderCheckboxList([{
+                name: 'isActive',
+                label: 'Active'
+            }, {
+                name: 'isPublic',
+                label: 'Public'
+            }, {
+                name: 'isAutoConfirm',
+                label: 'Confirmations'
+            }, {
+                name: 'notificationsEnabled',
+                label: 'Reminders Enabled'
+            }]),
+            ...this.renderFieldList([{
+                name: 'dayReminder',
+                label: 'Send Booking reminders X days before (single/day)'
+            }, {
+                name: 'weekReminder',
+                label: 'Send Booking reminders X days before (weekly)'
+            }, {
+                name: 'monthReminder',
+                label: 'Send Booking reminders X days before (monthly)'
+            }])
+        ], 'col3')
+    ];
+
+    renderModalContent = (fprops) => {
+        const formProps = {onSubmit: fprops.handleSubmit};
+        return (
+            <>
+                <Modal.Header>
+                    <Translate>Edit Room Details</Translate>
+                </Modal.Header>
+                <Modal.Content>
+                    <Form id="room-form" {...formProps}>
+                        <Grid columns={3}>
+                            {this.formDetails}
+                        </Grid>
+                    </Form>
+                </Modal.Content>
+                <Modal.Actions>
+                    <Button type="submit"
+                            form="room-form"
+                            primary>
+                        <Translate>Save</Translate>
+                    </Button>
+                </Modal.Actions>
+            </>
+        );
+    };
+
+    render() {
+        const {room} = this.state;
+        const props = {validate, onSubmit: this.handleSubmit};
+        if (!room) {
+            return <Dimmer active page><Loader /></Dimmer>;
+        }
+        return (
+            <Modal open onClose={this.handleCloseModal} size="large" closeIcon>
+                <FinalForm {...props}
+                           render={this.renderModalContent}
+                           initialValues={room} />
+            </Modal>
+        );
+    }
+}
+
+export default connect(
+    null, dispatch => ({
```

Comment:
linebreak after `null,` - otherwise it almost looks like you have an arrow function that accepts `(null, dispatch)`